arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -122,6 +124,14 @@ using namespace arangodb;
 using namespace arangodb::application_features;
 using namespace arangodb::options;
 
+namespace {
+using Env = rocksdb::Env;
+std::unique_ptr<rocksdb::Env> checksumEnv;
```

Comment:
Can we store the pointer to the env inside the RocksDBEngine? This would be much better than storing it in a static variable.